lib/editor/components/map/DirectionIconsLayer.js

```diff
@@ -35,14 +35,14 @@ export default class DirectionIconsLayer extends Component {
       const iconInterval = zoom > 15
         ? 200
         : zoom > 14
-        ? 500
-        : zoom > 12
-        ? 2000
-        : zoom > 10
-        ? 4000
-        : zoom > 6
-        ? 8000
-        : 10000
+          ? 500
+          : zoom > 12
+            ? 2000
+            : zoom > 10
+              ? 4000
+              : zoom > 6
+                ? 8000
+                : 10000
```

Comment:
I used to think these nested ternary operators were bad, but [this aricle](https://medium.com/javascript-scene/nested-ternaries-are-great-361bddd0f340) changed my mind.